src/Mcfedr/QueueManagerBundle/Manager/DoctrineDelayQueueManager.php

```diff
@@ -55,7 +55,7 @@ public function put(string $name, array $arguments = [], array $options = []): J
             $jobTime = new Carbon("+{$options['delay']} seconds", new \DateTimeZone('UTC'));
         }
 
-        if (!isset($jobTime) || $jobTime < new \DateTime('+30 seconds', new \DateTimeZone('UTC'))) {
+        if (!isset($jobTime) || $jobTime < new \DateTime('+5 seconds', new \DateTimeZone('UTC'))) {
```

Comment:
I think you need to add an option `array $options` in the constructor to configure this, with 30 remaining the default

Reply:
Yes, good idea. I thought about it because another manager can work with a delay of less than 30 seconds without DoctrineDelay.